### DISCUSSION TITLE
Overseer No Longer Displaying Movies Or Series

### DISCUSSION THREAD

**Discussion by brntn91** [sct/overseerr#3638] [Help]
I've had Overseer running for over a couple of months now with no problems, until about a week ago, movies and series stopped displaying, it just says "movie not found" with the TMDB ID number, I'm able to log in to Overseerr in and outside my own network.
I'm running on a Synology NAS with a reverse proxy, it was working fine even without proxy support, I've now turned that on, it hasn't made any difference.
Example of some of my logs:
[error][Plex Scan]: Failed to process Plex media{"errorMessage":"[TMDB] Failed to fetch TV show details: connect ETIMEDOUT
[debug][API]: Something went wrong retrieving popular series{"errorMessage":"[TMDB] Failed to fetch discover TV: connect ETIMEDOUT

**Comment by designeramic** [comment]
I also have Overseerr running in Docker on a Synology NAS. I'm having the exact same issue and I'm not sure what to do. Overseerr was working perfectly fine until about a week ago and I'm not sure what happened. I didn't make any changes to anything and it just suddenly stopped loading trending stuff. Getting same errors in my debug log on Portainer.

**Reply by designeramic** [reply]
In case this helps anyone: I forgot I was using the linuxserver image instead of the sct, so replaced my container with sct and everything works again. [v1.33.2-ls82](https://github.com/linuxserver/docker-overseerr/releases/tag/v1.33.2-ls82) seemed to be the issue.

**Reply by brntn91** [reply]
Mine resolved itself, no idea why it stopped, but it's back online again.

**Comment by mduijm** [comment]
with the cli command: 'docker exec -it [containerID] bash' you can go 'inside' the container, from there you can try to do some ping etc to check whether or not DNS and all is working. Perhaps good to use next time you run into issues. They seem network related.

**Comment by mduijm** [comment]
and with: 'docker ps' you get a list of containers running and their corresponding ID's.

**Comment by FiftyKnight** [comment]
This happened with me and it was a DNS problem caused by my VPN dropping with the kill switch on.  Fixed the VPN and it started working again.  Hope this helps someone.
